src/securityPolicy.c

```diff
@@ -276,9 +253,9 @@ security_policy_t policyForSignTxInit(
 		ASSERT(false);
 	}
 
-	WARN_IF(!is_tx_network_verifiable(numOutputs, numWithdrawals, txSigningMode));
+	DENY_UNLESS(isValidNetworkId(networkId));
```

Comment:
Since this is a `DENY` policy, it should be moved above the switch I think.

Reply:
Good eye, yes, we have a bunch of these lying around, especially in the calling side of the policies.